doc/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## 1.16.1
+
+Released on 27th May 2020
+
+Updated to @polkadot/api package v1.16.1
+Updated error logging for certain API calls with nested calls to other endpoints
```

Comment:
Remove `certain`

Reply:
Changed in commit 58eb1716605423364e9dd2b0286cb6162eff333f